.travis.yml

```diff
@@ -10,5 +10,4 @@ dist: trusty
 script:
  - make lint
```

Comment:
Follow the convention described [here](https://github.com/m3db/ci-scripts) for the makefile. 

use `make all` in the .travis.yml like other M3DB repos: https://github.com/m3db/m3db/blob/master/.travis.yml